client/components/settings/edit-access-modal.js

```diff
@@ -21,7 +27,9 @@ export default class EditAccessModal extends React.Component {
 
         let customFields;
         if (this.props.customFields) {
-            customFields = this.props.customFields.map(this.extractCustomFieldValue);
+            customFields = this.props.customFields.map(
+                (field, index) => this.extractCustomFieldValue(field, index)
```

Comment:
Here is the real fix. Without this, there is a browser error "this is undefiened" when extractCustomFieldValue is called